### PR TITLE
travis ci

### PR DESCRIPTION
**Pull request by hanabi1224**

### CODE REVIEW

**Comment by hanabi1224** [comment]
CI on OSX and linux
https://travis-ci.org/hanabi1224/Functional/builds/518613185?utm_source=github_status&utm_medium=notification   https://github.com/JohannesMoersch/Functional/issues/42

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+  only:
+    - master
+script:
+  - dotnet restore
+  - dotnet build -c Release
+  - dotnet test -c Release -v n
+#deploy:
+#  skip_cleanup: true
+#  provider: script
+#  script: dotnet nuget push ${TRAVIS_BUILD_DIR}/**/*.nupkg -k ${NUGET_API_KEY}
```
Comment (hanabi1224) [inline .travis.yml]: This will perform automatic nuget publish when version is updated

Comment (hanabi1224) [reply .travis.yml]: @JohannesMoersch I would suggest putting your encoded nuget api key here to enable automatic nuget publish

**Comment by JohannesMoersch** [comment]
Why Travis and Appveyor?

**Comment by hanabi1224** [comment]
@JohannesMoersch appveyor does not cover OSX : )  this travis setup tests both linux and osx

**Comment by JohannesMoersch** [comment]
Oh, okay. Why not use Travis CI for everything then?

**Comment by hanabi1224** [comment]
@JohannesMoersch dotnet support on travis is not official but community supoorted, not supporing windows. Appveyor is more convinient for things like  cd

**Comment by RyanMarcotte** [comment]
This pull request has seen no activity for quite some time, and with a new GitHub action build process in place it seems unlikely that this will be merged in.  Closing.

**Pull request closed without merging**